research/picai/prepare_data.py

```diff
@@ -273,24 +273,33 @@ def main() -> None:
     )
 
     args = parser.parse_args()
+
     if args.size is not None and len(args.size) != 3:
         raise ValueError("Argument size must have length 3")
-
-    if args.physical_size is not None and len(args.physical_size) != 3:
-        raise ValueError("Argument physical_size must have length 3")
+    size = (int(args.size(0)), int(args.size(1)), int(args.size(2))) if args.size else None
```

Comment:
Thanks for cleaning this up :)

Reply:
It was throwing a mypy error for me. Not really sure why it didn't throw on your PR but it's all good 🙂